asistente_ladm_col/logic/quality/polygon_quality_rules.py

```diff
@@ -356,47 +363,73 @@ def check_plot_nodes_covered_by_boundary_points(self, db):
 
-            buildings_with_no_plot, buildings_not_within_plot = self.qgis_utils.geometry.get_buildings_out_of_plots(layers[db.names.OP_BUILDING_T], layers[db.names.OP_PLOT_T], db.names.T_ID_F)
+            building_disjoin, building_overlaps, building_within = GeometryUtils.get_polygon_relation_polygon(layers[names.OP_BUILDING_T], layers[names.OP_PLOT_T])
+
+            tid_buildings = self.check_building_not_associate_correct_plot(building_within,
+                                                                           layers[names.OP_BUILDING_T],
+                                                                           layers[names.OP_PLOT_T],
+                                                                           layers[names.OP_PARCEL_T],
+                                                                           layers[names.COL_UE_BAUNIT_T],
+                                                                           layers[names.OP_CONDITION_PARCEL_TYPE_D],
+                                                                           names)
 
             new_features = list()
-            for building_with_no_plot in buildings_with_no_plot:
+            # Buildings that are not covered by a plot
+            for building_with_no_plot in layers[names.OP_BUILDING_T].getFeatures(building_disjoin):
                 new_feature = QgsVectorLayerUtils().createFeature(
                                 error_layer,
                                 building_with_no_plot.geometry(),
-                                {0: building_with_no_plot[db.names.T_ILI_TID_F],
+                                {0: building_with_no_plot[names.T_ILI_TID_F],
                                  1: self.quality_rules_manager.get_error_message(QUALITY_RULE_ERROR_CODE_E300901),
                                  2: QUALITY_RULE_ERROR_CODE_E300901})
                 new_features.append(new_feature)
 
-            for building_not_within_plot in buildings_not_within_plot:
+            # Buildings that are not within by a plot
+            for building_not_within_plot in layers[names.OP_BUILDING_T].getFeatures(building_overlaps):
                 new_feature = QgsVectorLayerUtils().createFeature(
                                 error_layer,
                                 building_not_within_plot.geometry(),
-                                {0: building_not_within_plot[db.names.T_ILI_TID_F],
+                                {0: building_not_within_plot[names.T_ILI_TID_F],
                                  1: self.quality_rules_manager.get_error_message(QUALITY_RULE_ERROR_CODE_E300902),
                                  2: QUALITY_RULE_ERROR_CODE_E300902})
                 new_features.append(new_feature)
             data_provider.addFeatures(new_features)
 
+            exp = "{} in ({})".format(names.T_ID_F, ", ".join([str(tid) for tid in tid_buildings]))
+            # Building is withing by a plot but this is not the associate in the topology table
```

Comment:
"within a plot"

That's not a topology table, that's only an intermediate table. 
"but this is not the corresponding plot"